testcontainers/testcontainers-rs
# docs: add scaffolding for testcontainers docs system

## What does this PR do?
It adds the base Mkdocs project for the testcontainers docs system.

It uses tc-go as reference, removing all the go-specific content but the general one.

## Why is it important?
Homogeneous and consistent docs system across testcontainers projects.

## How to test this PR
As described in the `docs/contributing_docs.md` file in this PR:

* Ensure that you have Python 3.8.0 or higher.
* Create a Python virtualenv. E.g. `python3 -m venv tc-venv`.
* Activate the virtualenv. E.g. `source tc-venv/bin/activate`.
* Run `pip3 install -r requirements.txt && ./tc-venv/bin/mkdocs serve` from the `testcontainers-rs` root directory. It will start a local auto-updating MkDocs server.

## Follow-ups
- Add features
- Add modules
- Create rust.testcontainers.org domain
- Enable netlify deployments and previews in PRs
